### ISSUE TITLE
[QUESTION] Is this project being actively maintained?

### ISSUE THREAD

**Issue by alexborton** [TheWidlarzGroup/react-native-notificated#187]
We really like this project and it ticks a lot of boxes for our Flash/Toast implementations, but it doesn't look like it's moved along in a while. Are you actively maintaining or planning to maintain this project?

**Comment by mateki0** [comment]
Yes it is, new version will be soon released. Are there any missing features that you would like to see or any bugs in current version?

**Comment by Sovent** [comment]
@mateki0 hi. I wonder if you are going to update react-native-gesture-handler dependency in the project. I have  "react-native-gesture-handler": "~2.8.0", in my package.json and I get 
```
Could not resolve dependency:
peer react-native-gesture-handler@"^1.10.3" from react-native-notificated@0.1.0
```
error trying to install this package. Should not it be >=1.10.3 ?

**Comment by Okelm** [comment]
Hi @alexborton Hi @Sovent ! Thanks for contributing. At this point we're working on the roadmap for the project, so if you have some ideas for future development, even some crazy once, we're happy to hear them :) Do you use the library currently?
